models/news.models.js

```diff
@@ -53,10 +53,32 @@ exports.fetchUsers = () => {
   });
 };
 
-exports.fetchArticle = () => {
+exports.fetchArticle = (sort_by = "created_at", order = "desc", topic) => {
+  const sortOptions = [
+    "title",
+    "article_id",
+    "topic",
+    "created_at",
+    "votes",
+    "author",
+    "comment_count",
+  ];
+  const orderOptions = ["asc", "desc"];
+  let topicStr = "";
+  if (!sortOptions.includes(sort_by)) {
+    return Promise.reject("Invalid sort query");
+  }
+  if (!orderOptions.includes(order)) {
+    return Promise.reject("Invalid order query");
+  } else {
+    order.toUpperCase();

```

Comment:
You don't need to uppercase the order options, it will work even if all in lowercase :)